packages/serverpod_serialization/lib/src/serialization.dart

```diff
@@ -83,6 +83,9 @@ abstract class SerializationManager {
         entity is double ||
         entity is SerializableEntity) {
       return '$entity';
+    } else if (entity is List<SerializableEntity>) {
+      // print('this is Seriziable List');
```

Comment:
Avoid leaving empty comments in the code

Reply:
ok